django/utils/lorem_ipsum.py

```diff
@@ -1,7 +1,6 @@
 """
 Utility functions for generating "lorem ipsum" Latin text.
 """
-
```

Comment:
Please don't make changes to this file in this PR. We don't want to mix refactoring and additional test coverage.